### PR TITLE
Endpoint details API

### PR DESCRIPTION
**Pull request by WilliamBZA**
## Description
This PR adds support for querying endpoint instance metrics data by logical endpoint name.

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Dependent on #30

File: src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs [hunk 1]
```diff
@@ -34,11 +43,35 @@ public static IEnumerable<MonitoredEndpoint> AggregateIntoLogicalEndpoints(Proce
...
+            var instanceIds = criticalTime.Concat(processingTime).Select(i => i.Id).Distinct();
+
+            foreach (var instanceId in instanceIds)
+            {
+                var monitoredInstance = new MonitoredEndpointInstance
+                {
+                    Id = instanceId.InstanceId,
+                    Name = instanceId.InstanceName,
+                    ProcessingTime = AggregateTimings(processingTime.Where(pt => pt.Id == instanceId).ToList()),
+                    CriticalTime = AggregateTimings(criticalTime.Where(ct => ct.Id == instanceId).ToList())
```
Comment (mikeminutillo) [inline src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs]: These will perform reference equality, even though we've overridden `Object.Equals`.

We should override the equality operator or use `.Equals()` here https://msdn.microsoft.com/ru-ru/library/ms173147(v=vs.80).aspx

**Comment by WojcikMike** [comment]
@mikeminutillo your comment was addressed, please merge

**Comment by WojcikMike** [comment]
I've rebased to develop

**Pull request merged**